src/app/components/dataset-details/dataset-details.component.html

```diff
@@ -12,12 +12,99 @@ <h3>Keywords</h3>
+            </div>
+            <div class="w-full">
+              <h4>URL:</h4>
+              <p><a href="{{ distribution.url }}" rel="noopener" target="_blank">{{ distribution.url }}</a></p>
+            </div>
+            <div class="w-full lg:w-1/2">
+              <h4>Issued:</h4>
+              <p>{{ getFormattedDate(distribution.issued) }}</p>
+            </div>
+            <div class="w-full lg:w-1/2">
+              <h4>Modified:</h4>
+              <p>{{ getFormattedDate(distribution.modified) }}</p>
+            </div>
+            <div class="w-full" *ngIf="distribution.access_url">
+              <h4>Access URL:</h4>
+              <p><a href="{{ distribution.access_url }}" rel="noopener" target="_blank">{{ distribution.access_url }}</a></p>
+            </div>
+            <div class="w-full" *ngIf="distribution.download_url">
+              <h4>Download URL:</h4>
+              <p><a href="{{ distribution.download_url }}" rel="noopener" target="_blank">{{ distribution.download_url }}</a></p>
+            </div>
+            <div class="w-full lg:w-1/2" *ngIf="distribution.access_services">
+              <h4>Access Services:</h4>
+              <p>{{ distribution.access_services }}</p>
+            </div>
+            <div class="w-fll lg:w-1/2" *ngIf="distribution.compress_format">
+              <h4>Compress Format:</h4>
+              <p>{{ distribution.compress_format }}</p>
+            </div>
+            <div class="w-full lg:w-1/2" *ngIf="distribution.availability">
+              <h4>Availability:</h4>
+              <p>{{ distribution.availability }}</p>
+            </div>
+            <div class="w-fll lg:w-1/2" *ngIf="distribution.language">
+              <h4>Language:</h4>
+              <p>{{ distribution.language }}</p>
+            </div>
+            <div class="w-full lg:w-1/2">
+              <h4>Rights:</h4>
+              <p>{{ distribution.rights }}</p>
+            </div>
+            <div class="w-full lg:w-1/2">
+              <h4>State:</h4>
+              <p>{{ distribution.state }}</p>
+            </div>
+            <div class="w-full lg:w-1/2" *ngIf="distribution.temporal_resolution">
+              <h4>Temporal Resolution:</h4>
+              <p>{{ distribution.temporal_resolution }}</p>
+            </div>
+            <div class="w-full lg:w-1/2" *ngIf="distribution.spatial_resolution_in_meters">
+              <h4>Spatial Resolution (meters):</h4>
+              <p>{{ distribution.spatial_resolution_in_meters }}</p>
+            </div>
+            <div class="w-full lg:w-1/2" *ngIf="distribution.cache_url">
+              <h4>Cache Url:</h4>
+              <p><a href="{{ distribution.cache_url }}" rel="noopener" target="_blank">{{ distribution.cache_url }}</a></p>
+            </div>
+            <div class="w-full lg:w-1/2" *ngIf="distribution.cache_last_updated">
+              <h4>Cache Last Updated:</h4>
+              <p>{{ getFormattedDate(distribution.cache_last_updated) }}</p>
```

Comment:
This things I see that there are mostly solved with a Pipe. For the sake of reusability and make code cleaner  

getFormattedDate becomes a pipe. And the code will be then for example 

`distribution.cache_last_updated | formattedDate`